x-pack/plugins/security_solution/public/resolver/view/panels/process_event_list.tsx

```diff
@@ -1,290 +1,290 @@
+    margin-right: 0.25em;
+  }
+
+  & .euiText {
+    display: inline;
+  }
+
+  & .euiText p {
+    display: inline;
+  }
+`;
+
+const DisplayList = memo(function DisplayList({
+  crumbs,
+  matchingEventEntries,
+  eventType,
+  processEntityId,
+}: {
+  crumbs: Array<{ text: string | JSX.Element; onClick: () => void }>;
+  matchingEventEntries: MatchingEventEntry[];
+  eventType: string;
+  processEntityId: string;
+}) {
+  const relatedLookupsByCategory = useSelector(selectors.relatedEventInfoByEntityId);
+  const lookupsForThisNode = relatedLookupsByCategory(processEntityId);
+  const shouldShowLimitWarning = lookupsForThisNode?.shouldShowLimitForCategory(eventType);
+  const numberDisplayed = lookupsForThisNode?.numberActuallyDisplayedForCategory(eventType);
+  const numberMissing = lookupsForThisNode?.numberNotDisplayedForCategory(eventType);
+
+  return (
+    <>
+      <StyledBreadcrumbs breadcrumbs={crumbs} />
+      {shouldShowLimitWarning && typeof numberDisplayed !== 'undefined' && numberMissing ? (
+        <StyledRelatedLimitWarning
+          eventType={eventType}
+          numberActuallyDisplayed={numberDisplayed}
+          numberMissing={numberMissing}
+        />
+      ) : null}
+      <EuiSpacer size="l" />
+      <>
+        {matchingEventEntries.map((eventView, index) => {
+          const { subject, descriptor = '' } = eventView.name;
+          return (
+            <Fragment key={index}>
+              <EuiText>
+                <BoldCode>
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.resolver.panel.relatedEventDetail.categoryAndType"
+                    values={{
+                      category: eventView.eventCategory,
+                      eventType: eventView.eventType,
+                    }}
+                    defaultMessage="{category} {eventType}"
+                  />
+                </BoldCode>
+                <StyledTime dateTime={eventView.formattedDate}>
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.resolver.panel.relatedEventDetail.atTime"
+                    values={{ date: eventView.formattedDate }}
+                    defaultMessage="@ {date}"
+                  />
+                </StyledTime>
+              </EuiText>
+              <EuiSpacer size="xs" />
+              <EuiButtonEmpty onClick={eventView.setQueryParams}>
+                <FormattedMessage
+                  id="xpack.securitySolution.endpoint.resolver.panel.processEventListByType.eventDescriptiveName"
+                  values={{ subject, descriptor }}
+                  defaultMessage="{descriptor} {subject}"
+                />
+              </EuiButtonEmpty>
+              {index === matchingEventEntries.length - 1 ? null : <EuiHorizontalRule margin="m" />}
+            </Fragment>
+          );
+        })}
+      </>
+    </>
+  );
+});
+
+export const ProcessEventList = memo(function ProcessEventList({
```

Comment:
:grey_question: I gave this a terrible name in retrospect. It should be called `relatedEventListForProcess`